python/ray/experimental/client/common.py

```diff
@@ -28,6 +28,9 @@ def binary(self):
     def from_remote_ref(cls, ref: ray_client_pb2.RemoteRef):
         return cls(id=ref.id, handle=ref.handle)
 
+    def __del__(self):
+        ray.call_release(self.id)
```

Comment:
Does this still work if we have multiple references to the same object on the client?